internal/mocks/aws-sdk/sts.go

```diff
@@ -0,0 +1,440 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
add the entry in the `doc.go` file similar we did here: https://github.com/mattermost/mattermost-cloud/blob/master/internal/mocks/aws-sdk/doc.go#L14-L15

Reply:
after you add just run `make mocks`  to update

Reply:
thanks done